agents/addon/extension/openai_chatgpt_python/openai_chatgpt_extension.py

```diff
@@ -73,11 +79,69 @@ def parse_sentence(sentence, content):
     return sentence, remain, found_punc
 
 
+def yuv420_to_rgb(yuv_data, width, height):
```

Comment:
Use [PIL convert](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.convert) could be simpler to achieve it? Alternatively, `RGBA` can be outputted by the `agora_rtc` extension, so no conversion is also good to go.